join.html

```diff
@@ -61,30 +61,29 @@ <h4>Fee Structure</h4>
     <table class="table">
       <tbody>
         <tr>
-          <td><strong>Backer Hacker</strong></td>
+          <td><strong>Standard Hacker</strong></td>
           <td>
             <ul>
-              <li>$100 per month, no further discounts.</li>
+              <li>$100 per month. Rates can be reduced to $60 per month by becoming a 'Volunteer Hacker.'</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Standard Hacker</strong></td>
+          <td><strong>Volunteer Hacker</strong></td>
           <td>
             <ul>
-              <li>$60 per month, or</li>
-              <li>$55 per month when paying 3 months or more in advance (up to 12 months)</li>
+              <li>$60 per month. To be on this rate you must complete 4 hours of volunteer work for HSBNE per month. Don't be shy! There's always plenty to do.</li>
             </ul>
           </td>
         </tr>
         <tr>
-          <td><strong>Starving Hacker</strong></td>
+          <td><strong>Starving Volunteer Hacker</strong></td>
```

Comment:
Should the names be changed to just "Member, Volunteer, Concessional Volunteer" or something? the last one could use a better name than my suggestion. Just wondering if theres value it keeping with the 'hacker' wording.